server/service/src/auth.rs

```diff
@@ -617,23 +612,60 @@ fn all_permissions() -> HashMap<Resource, PermissionDSL> {
     );
 
     map.insert(
-        Resource::MutateAsset,
+        Resource::AddAsset,
         PermissionDSL::Any(vec![
-            PermissionDSL::HasPermission(PermissionType::AssetMutate),
-            PermissionDSL::HasPermission(PermissionType::AssetMutateViaDataMatrix),
+            PermissionDSL::And(vec![PermissionDSL::Any(vec![
+                PermissionDSL::HasPermission(PermissionType::AssetMutate),
+                PermissionDSL::HasPermission(PermissionType::AssetMutateViaDataMatrix),
+            ])]),
+            PermissionDSL::HasStoreAccess,
+            PermissionDSL::HasPermission(PermissionType::EditCentralData),
+        ]),
+    );
+
+    map.insert(
+        Resource::EditAsset,
+        PermissionDSL::Any(vec![
+            PermissionDSL::And(vec![PermissionDSL::HasPermission(
+                PermissionType::AssetMutate,
+            )]),
```

Comment:
Since only one permission, could just go without the `vec`:

`PermissionDSL::HasPermission(PermissionType::AssetMutate),`

Same thing for `QueryAsset`

Reply:
It's not one permission tho..